index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Video App</title>
+  <link rel="stylesheet" href="./index.css" />
+</head>
+
+<body>
+
+  <div class="main">
+
+    <nav class="navbar">
+      <div class="brand">
+        <img src="./img/menu.png" alt="menu-icon" />
+        <span>Video App</span>
+      </div>
+
+      <div class="input-field">
+        <input class="searchBar" type="text" />
+        <img src="./img/search.png" alt="search-icon" />
+      </div>
+
+      <div class="menu-section">
+        <ul class="top-menu">
+
+          <li><a href=""><img src="./img/notification.png" alt="" /></a></li>
+          <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
+          <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
+          </li>
+          
+        </ul>
+
+      </div>
+    </nav>
+
+    <section class="main-section">
+
+      <div class="r-side">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/oGYv3sCuxVQ" title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+
+        <div class="title">
+          <h3>What does Earth sound like from space?</h3>
+        </div>
+
+        <div class="icon-bar">
+          <span class="icon-cols"> Like </span>
+          <span class="icon-cols"> Dislike </span>
+          <span class="icon-cols"> Save </span>
+          <span class="icon-cols"> Share </span>
+          <span class="icon-cols"> More </span>
+        </div>
```

Comment:
Use list instaed of span tags

Reply:
-I think It will be better to phrase your comments politely, as suggestions instead of orders.
For example:
This is a polite suggestion: "I think it would be better to use a list instead of span tags"

- It is also essential to state reasons/explanations on why a requested change should be implemented
For example:
"I think it would be better to use list instead of tags because list tags . . . . ."

- Adding emojis will make your suggestion seem friendly as well 🤗 

- After the requested change, you can ask the student what they think about your suggestion. 
For example:
"What do you think about these suggestions above? 💭 "


_Why don't we edit your comments to meet all the requirements I have just stated above??_ 🤔